gpytorch/mlls/exact_marginal_log_likelihood.py

```diff
@@ -59,9 +62,17 @@ def forward(self, function_dist, target, *params):
 
         # Get the log prob of the marginal distribution
         output = self.likelihood(function_dist, *params)
-        res = output.log_prob(target)
-        res = self._add_other_terms(res, params)
+        res = output.log_prob(target, combine_terms=self.combine_terms)
```

Comment:
Stylistically, the proposed change from `log_prob` to `_log_prob`  is problematic here, because you would essentially be calling a "private" method publicly. More generally I think the `combine_terms` option is broadly useful and burying it inside the class makes it harder to use. 

Personally I don't see why the GPyTorch `log_prob` API can't allow optional keyword arguments like `combine_terms`, as long as the default behavior is consistent.

Reply:
@gpleiss @jacobrgardner care to weigh in?

Reply:
a compromise would be to just call it  `log_prob_terms` instead of `_log_prob`